9.4-jre8/alpine/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM openjdk:8-jre-alpine
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN addgroup -S jetty && adduser -D -S -H -G jetty jetty && rm -rf /etc/group- /etc/passwd- /etc/shadow-
+
+ENV JETTY_HOME /usr/local/jetty
+ENV PATH $JETTY_HOME/bin:$PATH
+RUN mkdir -p "$JETTY_HOME"
+WORKDIR $JETTY_HOME
+
+ENV JETTY_BASE /var/lib/jetty
+RUN mkdir -p "$JETTY_BASE"
+
+ENV JETTY_VERSION 9.4.1.v20170120
+ENV JETTY_TGZ_URL https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-home/$JETTY_VERSION/jetty-home-$JETTY_VERSION.tar.gz
+
+# GPG Keys are personal keys of Jetty committers (see https://dev.eclipse.org/mhonarc/lists/jetty-users/msg05220.html)
+ENV JETTY_GPG_KEYS \
+	# 1024D/8FB67BAC 2006-12-10 Joakim Erdfelt <joakime@apache.org>
+	B59B67FD7904984367F931800818D9D68FB67BAC \
+	# 4096R/438A3B7D 2016-08-15 Jesse McConnell (signing key) <jesse.mcconnell@gmail.com>
+	2A684B57436A81FA8706B53C61C3351A438A3B7D
+
+RUN set -xe \
+	# Install required packages for build time. Will be removed when build finishes.
+	&& apk add --no-cache --virtual .build-deps gnupg coreutils curl \
+
+	&& curl -SL "$JETTY_TGZ_URL" -o jetty.tar.gz \
+	&& curl -SL "$JETTY_TGZ_URL.asc" -o jetty.tar.gz.asc \
+	&& export GNUPGHOME="$(mktemp -d)" \
+	&& for key in $JETTY_GPG_KEYS; do \
+		gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; done \
+	&& gpg --batch --verify jetty.tar.gz.asc jetty.tar.gz \
+	&& rm -r "$GNUPGHOME" \
+	&& tar -xvzf jetty.tar.gz \
+	&& mv jetty-home-$JETTY_VERSION/* ./ \
+	&& sed -i '/jetty-logging/d' etc/jetty.conf \
+	&& rm jetty.tar.gz* \
+	&& rm -fr jetty-home-$JETTY_VERSION/ \
+
+	# Get the list of modules in the default start.ini and build new base with those modules, then add setuid
+	&& cd $JETTY_BASE \
+	&& modules="$(grep -- ^--module= "$JETTY_HOME/start.ini" | cut -d= -f2 | paste -d, -s)" \
```

Comment:
Looks like we should drop this `modules` line as well as the comment above, right?